content/en/Product Updates/release-notes-july-2024.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Release Notes: July 2024"
+linkTitle: "July 2024"
+weight: 45
+description: >-
+     Explore What's New from Cobalt This Month
+---
+
+{{% pageinfo %}} 
+Stay Up-to-Date with the Latest Enhancements on the Cobalt Platform. 
+{{% /pageinfo %}}
+
+### July 2024
+
+
+#### Attack Surface Updates
+
+In response to customer feedback and to better align with the comprehensive capabilities of the feature, we are excited to announce that 'Domains' has now been revamped as 'Attack Surface.' This new name reflects the broader scope and enhanced functionality that this feature offers to our users.
+
+As part of this update, we are introducing the all-new 'Attack Surface Insights' feature. We understand that managing multiple domains and hosts can be challenging for our customers, making it crucial to prioritize effectively. The new insights feature aims to simplify this process by providing high-level insights, enabling users to identify key focus areas and prioritize tasks seamlessly.
+
+With intuitive charts and visual aids, customers can now gain a consolidated overview of the health status of their attack surface, empowering them to make informed decisions and optimize their security strategies. This update underscores our commitment to helping our users achieve a more robust and proactive security posture.
+
+![Attack Surface Insights](/release-notes/Attack-Surface-Insights.png "Attack Surface Insights")
+
+
+---
+
+#### Ticketing Integrations Released for DAST Scans
+
+Customers have expressed a strong desire to seamlessly integrate their DAST findings directly into their ticketing systems, a highly requested feature for DAST. In response to this feedback, we have enhanced our public API and Integration Builder to empower customers to:
+
+- Retrieve their DAST targets data
+- Access their DAST scans data
+- Retrieve their DAST findings data
+
+With these improvements, customers will now receive real-time notifications when a DAST finding is detected or resolved. Additionally, we have introduced recipe templates in the Integration Builder Library, allowing customers to effortlessly push their DAST findings to popular platforms such as Jira Cloud, GitHub Issues, and Azure DevOps Boards.
+
+## Steps to Integrate with Ticketing Systems
```

Comment:
sorry, last thing - i think this title could be reduced a bit in size. it's larger than the other titles on the page

Reply:
agree - went in and made it smaller (##### instead of ##)